lib/inspec/resources/wmi.rb

```diff
@@ -95,6 +95,10 @@ def params
       @content
     end
 
+    def resource_id
```

Comment:
Actually, I disagree with this change. InSpec 4 will never support resource ID's - that would be backporting a feature to InSpec 4. The fix on this PR should be to remove the test for resource_id.